protofsm/state_machine.go

```diff
@@ -204,6 +206,12 @@ func (s *StateMachine[Event, Env]) Stop() {
 //
 // TODO(roasbeef): bool if processed?
 func (s *StateMachine[Event, Env]) SendEvent(event Event) {
+	log.Debugf("FSM(%v): sending event: %v", s.cfg.Env.Name(),
+		lnutils.NewLogClosure(func() string {
+			return spew.Sdump(event)
+		}),
```

Comment:
Theres `SpewLogClosure` specifically for this.

Reply:
Ah cool, was able to cut down quite a bit of chars using this.